uk.co.compucorp.civicrm.hrcore/gulp/tasks/backstopjs.js

```diff
@@ -1,64 +1,62 @@
-var _ = require('lodash');
-var argv = require('yargs').argv;
-var backstopjs = require('backstopjs');
-var clean = require('gulp-clean');
-var Chromy = require('chromy');
-var execSync = require('child_process').execSync;
-var file = require('gulp-file');
-var fs = require('fs');
-var gulp = require('gulp');
-var notify = require('gulp-notify');
-var path = require('path');
-var Promise = require('es6-promise').Promise;
-
-var utils = require('../utils');
-
-var BACKSTOP_DIR = path.join(__dirname, '..', '..', 'backstop_data');
-var CHROMY_STARTING_PORT = 9222;
-var DEFAULT_USER = 'civihr_admin';
-var USERS = ['admin', 'civihr_admin', 'civihr_manager', 'civihr_staff'];
-var CONFIG_TPL = { 'url': 'http://%{site-host}' };
-var FILES = {
+const _ = require('lodash');
+const argv = require('yargs').argv;
+const backstopjs = require('backstopjs');
+const clean = require('gulp-clean');
+const execSync = require('child_process').execSync;
+const file = require('gulp-file');
+const fs = require('fs');
+const gulp = require('gulp');
+const notify = require('gulp-notify');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const Promise = require('es6-promise').Promise;
+
+const utils = require('../utils');
+
+const BACKSTOP_DIR = path.join(__dirname, '..', '..', 'backstop_data');
```

Comment:
Please move `__dirname` to a separate variable, smth like `CURRENT_DIRECTIORY_PATH`

Reply:
`__dirname` is already a constant, i don't see the benefit of doing that (and devs should already know what it means anyway)